### ISSUE TITLE
Azure Striker Gunvolt - Striker Pack -: BIN and FNT Files

### ISSUE THREAD

**Issue by Sakaki15** [IcySon55/Kuriimu#318]
Issue: feature request
Console: Nintendo 3DS
Format extension(s): .FNT
Type: [archive]
Game Name: Azure Striker Gunvolt - Striker Pack - 

First 8 byte of file(s):
.FNT - [04 00 00 00 18 00 00 00 50 B9 00 00 0C C3 00 00] - [49 kB]

More details:
I understand FNT files are not text per se but bitmap fonts indicating the font and / or characters the game employes.

However, as much I as research I have trouble finding a consistent way to output them as PNG or figure out how other fan-game translation projects work upon these files to translate them to English. 

I'd like to use them for a translation project but it's not urgent so I don't mind waiting, I understand you must be very busy.

Thank you very much in advance. 

Sample files (if possible):
https://drive.google.com/open?id=0BzPTj5lm0oCOMnpjNXY3UklBbHM (FNT file)

**Comment by onepiecefreak3** [comment]
Link doesn't get me to files. Please repair.
And the bin files doesn't contain text so far as we can see.

**Comment by Sakaki15** [comment]
I changed the link and removed the BIN file since, like you said, it has no text.

Now I reshaped my issue to ask about how you normally work with these bitmap files (if this is the correct name).

Again, I apologize for any inconvenience caused but as much as I've tried to research on my own I'm having trouble finding a consistent method to edit game scripts since each game uses different formats / systems.
